WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Product Details/AggregatedProductListViewController.swift

```diff
@@ -0,0 +1,148 @@
+        view.backgroundColor = .listBackground
+    }
+
+    /// Setup: TableView
+    ///
+    func configureTableView() {
+        tableView.backgroundColor = .listBackground
+        tableView.estimatedRowHeight = Constants.rowHeight
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.registerNib(for: PickListTableViewCell.self)
+    }
+}
+
+
+// MARK: - UITableViewDataSource Conformance
+//
+extension AggregatedProductListViewController: UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let item = itemAtIndexPath(indexPath)
+        let product = lookUpProduct(by: item.productOrVariationID)
+        let addOns = itemAddOnsAttributes(item: item)
+        let itemViewModel = ProductDetailsCellViewModel(aggregateItem: item,
+                                                        currency: viewModel.order.currency,
+                                                        product: product,
+                                                        hasAddOns: addOns.isNotEmpty)
+        let cell = tableView.dequeueReusableCell(PickListTableViewCell.self, for: indexPath)
+        cell.selectionStyle = .default
+        cell.configure(item: itemViewModel, imageService: imageService)
+        cell.onViewAddOnsTouchUp = { [weak self] in
+            self?.itemAddOnsButtonTapped(addOns: addOns)
+        }
+
+        return cell
+    }
+}
+
+
+// MARK: - UITableViewDelegate Conformance
+//
+extension AggregatedProductListViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+
+        let orderItem = itemAtIndexPath(indexPath)
+        productWasPressed(orderItem: orderItem)
+    }
+}
+
+
+// MARK: - Private Helpers
+//
+private extension AggregatedProductListViewController {
+
+    func itemAtIndexPath(_ indexPath: IndexPath) -> AggregateOrderItem {
+        return items[indexPath.row]
+    }
+
+    func lookUpProduct(by productID: Int64) -> Product? {
+        return products?.filter({ $0.productID == productID }).first
+    }
+
+    /// Returns the item attributes that can be identified as add-ons attributes.
+    /// If the "view add-ons" feature is disabled an empty array will be returned.
+    ///
+    func itemAddOnsAttributes(item: AggregateOrderItem) -> [OrderItemAttribute] {
+        guard let product = lookUpProduct(by: item.productID), viewModel.dataSource.showAddOns else {
+            return []
+        }
+
+        let globalAddOns = viewModel.dataSource.addOnGroups
+        return AddOnCrossreferenceUseCase(orderItemAttributes: item.attributes, product: product, addOnGroups: globalAddOns).addOnsAttributes()
+    }
```

Comment:
I think we should not show the "View Add-ons" button on the shipped items screen because we can't really know if a shipped item contained an add-on.

Could we remove all the code associated with it?

<img width="404" alt="Screen Shot 2021-06-17 at 8 43 57 AM" src="https://user-images.githubusercontent.com/562080/122411644-87ee4100-cf4a-11eb-85d4-da2b90596d7b.png">